src/wscript_build

```diff
@@ -1,17 +1,95 @@
+# Detect a Windows build.
+is_win = 'WIN32' in ' '.join(bld.env.keys())
+
+# Detect MSVC
+is_msvc = bld.env.CXX_NAME == 'msvc'
+
 all_sources = bld.path.ant_glob('*.cpp scene-ideas/*.cc scene-terrain/*.cpp')
 common_sources = [f for f in all_sources if f.name.find('canvas-') == -1 and
                                             f.name.find('android') == -1 and
                                             f.name.find('native-state-') == -1 and
                                             f.name.find('gl-state-') == -1 and
                                             f.name.find('main.cpp') == -1]
-common_uselibs = ['libpng']
+local_uselibs = ['libpng-local', 'zlib-local', 'libjpeg-turbo-local']
+
+common_uselibs = []
 common_defines = ['USE_EXCEPTIONS']
 
+if is_win:
+  common_uselibs += local_uselibs
```

Comment:
We can append the [*-local] list directly here, no need for the local_uselibs variable (see comment further down).

Reply:
Also I think it's worth agregrating all the win vs linux specific differences in one "if" statement (e.g. below with the platform_libs).

Reply:
Done. Added platform_uselibs to all_uselibs. There's a few non-local libs in all_uselibs but I think that should be okay.